js/borrower-speak.js

```diff
@@ -0,0 +1,7 @@
+// https://www.codewars.com/kata/borrower-speak/train/javascript
+
+function borrow(s){
+  return s.replace(/[!@#$%^&?*;,:.]/g, "").replace(/\s/g, '').toLowerCase();
```

Comment:
When it comes to regexps, since they do not offer any kind of naming/comments, it's usually worth it to extract them to variables and provide a suitable name for the variable so it's clear what the regexp is.

For instance, the current regexp could be extracted to `specialCharactersRegexp`